docs/packages/pkg/flatbuffers.rst

```diff
@@ -10,9 +10,21 @@ flatbuffers
 ===========
 
 -  `Official GitHub <https://github.com/google/flatbuffers>`__
--  `Hunterized <https://github.com/hunter-packages/flatbuffers>`__
 -  `Example <https://github.com/ruslo/hunter/blob/develop/examples/flatbuffers/foo.cpp>`__
 
+.. code-block:: cmake
+
+    hunter_add_package(flatbuffers)
+    find_package(Flatbuffers CONFIG REQUIRED)
```

Comment:
So they use `FlatbuffersConfig.cmake` but imported namespace is `flatbuffers::` and target is `flatbuffers`. Strange choice.